docs/operation/training.rst

```diff
@@ -0,0 +1,237 @@
+
+.. code-block:: bash
+
+    git clone https://github.com/shantanuparab-tr/act.git act_training_evaluation
+
+
+Clone ACT++ if using Aloha Mobile
+
+.. code-block:: bash
+
+    git clone https://github.com/shantanuparab-tr/act_plus_plus.git act_training_evaluation
+
+
+Build and Install ACT Models
+===================================
+
+.. code-block:: bash
+   :emphasize-lines: 4
+
+    ├── act
+    │   ├── assets
+    │   ├── constants.py
+    │   ├── detr
+    │   ├── ee_sim_env.py
+    │   ├── imitate_episodes.py
+    │   ├── __init__.py
+    │   ├── policy.py
+    │   ├── record_sim_episodes.py
+    │   ├── scripted_policy.py
+    │   ├── sim_env.py
+    │   ├── utils.py
+    │   └── visualize_episodes.py
+    ├── COLCON_IGNORE
+    ├── conda_env.yaml
+    ├── LICENSE
+    └── README.md
+
+
+Navigate to the ``detr`` directory inside the repository and install the detr module whihc contains the model definitions using the below command:
+
+.. code-block:: bash
+
+    cd /path/to/act/detr && pip install -e .
+
+Training
+=============
+
+To start the training, follow the steps below:
+
+1. **Sanity Check**: 
+
+Ensure you have all the hdf5 episodes located in the correct folder after following the data collection steps :ref:`operation/data_collection:Task Creation`.
+
+2. **Source ROS Environment**:
+
+   .. code-block:: bash
+
+       source /opt/ros/humble/setup.bash
+       source interbotix_ws/install/setup.bash
+
+3. **Activate Virtual Environment**:
+
+   .. code-block:: bash
+
+       source act/bin/activate
+
+4. **Start Training**:
+
+   .. code-block:: bash
+
+       cd repo/act/
+       python3 imitate_episodes.py \
+       --task_name aloha_stationary_dummy \
+       --ckpt_dir <ckpt dir> \
+       --policy_class ACT \
+       --kl_weight 10 \
+       --chunk_size 100 \
+       --hidden_dim 512 \
+       --batch_size 8 \
+       --dim_feedforward 3200 \
+       --num_epochs 2000 \
+       --lr 1e-5 \
+       --seed 0
+
+.. note::
+
+   - ``task_name`` argument should match one of the task names in the ``TASK_CONFIGS``, as configured in the :ref:`operation/data_collection:Task Creation` section.
+   - ``ckpt_dir``: The relative location where the checkpoints and best policy will be stored.
+   - ``policy_class``: Determines the choice of policy 'ACT'/'CNNMLP'.
+   - ``kl_weight``: Controls the balance between exploration and exploitation.
+   - ``chunk_size``: Determines the length of the action sequence. K=1 is no action chunking and K=episode length is full open loop control.
+   - ``batch_size``: Low batch size leads to better generalization and high batch size results in slower convergence but faster training time.
+   - ``num_epochs``: Too many epochs lead to overfitting; too few epochs may not allow the model to learn.
+   - ``lr``: Higher learning rate can lead to faster convergence but may overshoot the optima, while lower learning rate might lead to slower but stable optimization.
+
+We recommend the following parameters:
```

Comment:
This whole can be a tip: https://docutils.sourceforge.io/docs/ref/rst/directives.html#tip

Thoughts?